examples/openai/.gitignore

```diff
@@ -0,0 +1,2 @@
+lib/
+node_modules/
```

Comment:
is this necessary? I think they should probably be included in the top level of the repo (if not already in main branch feel free to add it there)